packages/server/src/service-manager.ts

```diff
@@ -6,7 +6,7 @@ import { PageConfig, URLExt } from '@jupyterlab/coreutils';
 import { IServiceWorkerManager, WORKER_NAME } from './tokens';
 
 const VERSION = PageConfig.getOption('appVersion');
-
+const SW_PING_ENDPOINT = '/api/keep-sw-alive';
```

Comment:
Just a nitpicky comment

```suggestion
const SW_PING_ENDPOINT = '/api/service-worker-heartbeat';
```